packages/perseus-editor/src/__stories__/editor-page.stories.tsx

```diff
@@ -128,6 +128,14 @@ export const MafsWithLockedFiguresCurrent = (): React.ReactElement => {
     );
 };
 
+MafsWithLockedFiguresCurrent.parameters = {
+    chromatic: {
+        // Disabling because this isn't visually testing anything on the
+        // initial load of the editor page.
+        disable: true,
```

Comment:
What does this do? Note that we currently only run visual snapshot tests for files named `*regression.stories.tsx` ([configured here](https://github.com/Khan/perseus/blob/18be26f35f735e6ec985d65d48d9859e5227fa39/.github/workflows/storybook.yml#L47)). This file won't be tested on Chromatic with our current setup.

Reply:
🤔  I disabled this story because I noticed it show up in the chromatic "UI tests" check in the last Version Packages PR. I'm pretty sure that means it's still being run. Is it possible that there's some place where it's not being restricted properly?

Reply:
I haven't figured out why, but the number on the UI Tests check is not the actual number of tests that run. E.g. on this PR it says "350 tests unchanged"

<img width="305" alt="Screen Shot 2024-06-12 at 3 14 01 PM" src="https://github.com/Khan/perseus/assets/693920/43b253b4-695d-42b3-9c6d-cb86eab442be">

but if you click through to Chromatic, it shows that 350 stories were _uploaded_, and only 48 visual tests were run.

<img width="653" alt="Screen Shot 2024-06-12 at 3 14 23 PM" src="https://github.com/Khan/perseus/assets/693920/d8064d98-1571-4e28-9f60-fe3651a82e3d">

I'm pretty sure the numbers on Chromatic.com are correct, because if we were really running 350 snapshot tests on every PR, we'd run out of monthly quota for our current (free) Chromatic plan.

Reply:
(Discussed offline.) Definitely something weird going on 🤔 I'll keep an eye out and point it out next time it happens. In the meantime, it doesn't hurt to keep this story disabled on Chromatic, so I'll leave this as is.

Thanks Ben!